backend/coreapp/views/scratch.py

```diff
@@ -62,6 +62,7 @@ def compile_scratch(scratch: Scratch) -> CompilationResult:
         scratch.compiler_flags,
         scratch.source_code,
         scratch.context,
+        scratch.diff_label,
```

Comment:
Is this an artifact of the previous merge mess?

Reply:
No, this is one of the changes I had to make for the compiler to work. The diff label has to be defined for the asm diffing process to work with the compiler.